enki/PhysicalEngine.cpp

```diff
@@ -225,14 +225,15 @@ namespace Enki
 	
 	const double PhysicalObject::g = 9.81;
 	
-	PhysicalObject::PhysicalObject(void) :
+	PhysicalObject::PhysicalObject(bool mouvable) :
```

Comment:
Also, this is a breakage of encapsulation, as the _movability_ is a viewer parameter, while the core of Enki is viewer-independant.